packages/node_modules/nav-frontend-lukknapp/md/Lukknapp.ingress.mdx

```diff
@@ -0,0 +1,6 @@
+import Alertstripe from 'nav-frontend-alertstriper';
```

Comment:
Tenker kanskje at hele `nav-frontend-lukknapp` pakken bare kan slettes her? 🤔

Reply:
Hvilke støtte er det ønskelig at designsystemet skal gi for legacy-komponenter? Er det evt en ide å ha en egen plassering for legacy-komponenter?